tools/sgen/input/FluentUIStyle.yml

```diff
@@ -346,3 +346,17 @@ AP_MSFDrawerTokens:
   cornerRadius: $Border.radius.xxLarge
   verticalShadowOffset: $Spacing.xSmall
   horizontalShadowOffset: $Spacing.small
+
+AP_MSFPersonaGridItemTokens:
+  avatarInterspace: [ small: $Spacing.xSmall, large: $Spacing.small ]
+  backgroundColor: $Colors.Background.neutral1
+  bottomPadding: $Spacing.xSmall
+  labelColor: $Colors.Foreground.neutral1
+  labelFont: [ small: $Typography.caption1, large: $Typography.subheadline ]
+  sublabelColor: $Colors.Foreground.neutral3
+  sublabelFont: $Typography.footnote
+  topPadding: $Spacing.xSmall
+
+AP_MSFPersonaGridTokens:
```

Comment:
Should GridItem extend from Grid?

Reply:
Good question! I'm picturing it as a compositional model: the `Grid` is actually the full collection of faces, while each `GridItem` maps to an individual persona.

I'll see if I can think of any better names to make that relationship clearer, but I'm also open to suggestions!

Reply:
I think it makes sense to keep them as separate APs (like the AP_MSFListTokens and the AP_MSFListCellTokens).
The Grid can't exist without the GridItem, but the GridItem can be used as a standalone control.